docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+services:
+  db:
+    image: 'postgres:latest'
+    container_name: postgres_fastapi
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - mate_fastapi_data:/var/lib/postgresql/data/
+    networks:
+      - fastapi_network
+
+  pgadmin:
+    image: dpage/pgadmin4
+    container_name: pgadmin_fastapi
+    ports:
+      - "3333:80"
+    env_file:
+      - .env
+    links:
+      - db:db
+    depends_on:
+      - db
+    volumes:
+      - fastapi_pgadmin_data:/var/lib/pgadmin
+    networks:
+      - fastapi_network
+
+  web:
+    restart: always
+    build: .
+    container_name: backend_fastapi
+    command: ["/entrypoint.sh"]
+    environment:
+      - WATCHFILES_FORCE_POLLING=true
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - .:/usr/src/fastapi
+    links:
+      - db:db
+    depends_on:
+      - db
+    networks:
+      - fastapi_network
+
+  migrator:
+    build: .
+    container_name: alembic_migrator
+    command: /migration_script.sh
```

Comment:
Ensure that the `/migration_script.sh` path is correct and that the script is executable. This script should handle the database migrations using Alembic.